### PR TITLE
Incorrect records due to extra space in the URLs #5275

### PR DESCRIPTION
**Pull request by RAJU2529**
the following domain address blank gaps are fixed
ussuk1southprod. blob.core.windows.net
ussuk1westprod. blob.core.windows.net
wsuk1southprod. blob.core.windows.net
wsuk1westprod. blob.core.windows.net

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Markdown style checks needed for this PR. Thanks.

**Comment by illfated** [comment]
Line 114 seems to have been split into 2 lines, 114 (`United Kingdom | ```uk.vortex-win.data.microsoft.com``` <br>```uk-v20.events.data.microsoft.com``` <br>`) and 115 (` ```ussuk1southprod.blob.core.windows.net``` <br>```ussuk1westprod.blob.core.windows.net``` <br>```winatp-gw-uks.microsoft.com``` <br>```winatp-gw-ukw.microsoft.com``` <br>```wsuk1southprod.blob.core.windows.net``` <br>```wsuk1westprod.blob.core.windows.net``` `).

Do you think you can undo that line break and merge back the lines into only 114, so the Rich Text Diff view will be able to show the content correctly?

File: windows/security/threat-protection/microsoft-defender-atp/configure-proxy-internet.md [hunk 1]
```diff
@@ -111,7 +111,8 @@ If a proxy or firewall is blocking all traffic by default and allowing only spec
 -|-
 Common URLs for all locations | ```crl.microsoft.com```<br> ```ctldl.windowsupdate.com``` <br>```events.data.microsoft.com```<br>```notify.windows.com```<br> ```settings-win.data.microsoft.com```
 European Union | ```eu.vortex-win.data.microsoft.com``` <br> ```eu-v20.events.data.microsoft.com``` <br> ```usseu1northprod.blob.core.windows.net```  <br>```usseu1westprod.blob.core.windows.net``` <br> ```winatp-gw-neu.microsoft.com``` <br> ```winatp-gw-weu.microsoft.com``` <br>```wseu1northprod.blob.core.windows.net``` <br>```wseu1westprod.blob.core.windows.net``` 
-United Kingdom | ```uk.vortex-win.data.microsoft.com``` <br>```uk-v20.events.data.microsoft.com``` <br>```ussuk1southprod. blob.core.windows.net``` <br>```ussuk1westprod. blob.core.windows.net``` <br>```winatp-gw-uks.microsoft.com``` <br>```winatp-gw-ukw.microsoft.com``` <br>```wsuk1southprod. blob.core.windows.net``` <br>```wsuk1westprod. blob.core.windows.net``` 
+United Kingdom | ```uk.vortex-win.data.microsoft.com``` <br>```uk-v20.events.data.microsoft.com``` <br>
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/configure-proxy-internet.md]:
```suggestion
United Kingdom | ```uk.vortex-win.data.microsoft.com``` <br>```uk-v20.events.data.microsoft.com``` <br>```ussuk1southprod.blob.core.windows.net``` <br>```ussuk1westprod.blob.core.windows.net``` <br>```winatp-gw-uks.microsoft.com``` <br>```winatp-gw-ukw.microsoft.com``` <br>```wsuk1southprod.blob.core.windows.net``` <br>```wsuk1westprod.blob.core.windows.net``` 
```

File: windows/security/threat-protection/microsoft-defender-atp/configure-proxy-internet.md [hunk 2]
```diff
@@ -111,7 +111,8 @@ If a proxy or firewall is blocking all traffic by default and allowing only spec
...
+```ussuk1southprod.blob.core.windows.net``` <br>```ussuk1westprod.blob.core.windows.net``` <br>```winatp-gw-uks.microsoft.com``` <br>```winatp-gw-ukw.microsoft.com``` <br>```wsuk1southprod.blob.core.windows.net``` <br>```wsuk1westprod.blob.core.windows.net```
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/configure-proxy-internet.md]:
```suggestion
```

**Comment by illfated** [comment]
Very well, the Rich Text Diff view has been fixed to show the content properly.
Then again, we need to add HTML line breaks to align the text properly, the way it was meant to be presented on the page.

Also, before I forget this, we need to reference the issue ticket page here in the comments, so it can be linked and shown on the ticket page:
- **Incorrect records due to extra space in the URLs** #5275

**Pull request merged**